internal/cluster/cluster.go

```diff
@@ -1,15 +1,38 @@
 package cluster
 
+import (
+	"github.com/buildkite/go-buildkite/v3/buildkite"
+)
+
 type Cluster struct {
-	OrganizationSlug string
-	ClusterID        string
-	Queues           []Queue
-	Name             string
-	Description      string
+	Color           *string                  `json:"color,omitempty"`
+	CreatedAt       buildkite.Timestamp      `json:"created_at"`
+	CreatedBy       buildkite.ClusterCreator `json:"created_by"`
+	DefaultQueueID  *string                  `json:"default_queue_id"`
+	DefaultQueueURL *string                  `json:"default_queue_url"`
+	Description     *string                  `json:"description"`
+	Emoji           *string                  `json:"emoji"`
+	GraphQLID       string                   `json:"graphql_id"`
+	ID              string                   `json:"id"`
+	Name            string                   `json:"name"`
+	Queues          []ClusterQueue
+	QueuesURL       string `json:"queues_url"`
+	URL             string `json:"url"`
+	WebURL          string `json:"web_url"`
 }
 
-type Queue struct {
-	Id           string
-	Name         string
-	ActiveAgents int
+type ClusterQueue struct {
+	ActiveAgents       int
+	ClusterID          string                    `json:"cluster_id"`
+	CreatedAt          buildkite.Timestamp       `json:"created_at"`
+	CreatedBy          buildkite.ClusterCreator  `json:"created_by"`
+	Description        *string                   `json:"description"`
+	DispatchPaused     bool                      `json:"dispatch_paused"`
+	DispatchPausedAt   *buildkite.Timestamp      `json:"dispatch_paused_at"`
+	DispatchPausedBy   *buildkite.ClusterCreator `json:"dispatch_paused_by"`
+	DispatchPausedNote *string                   `json:"dispatch_paused_note"`
+	ID                 string                    `json:"id"`
+	Key                string                    `json:"key"`
+	URL                string                    `json:"url"`
+	WebURL             string                    `json:"web_url"`
 }
```

Comment:
Wait that seems passive-aggressive.

What I more mean is; do you have future plans/thoughts on how it would benefit to have these defined here or something like that?

Reply:
lol tbh I just added them kind of blind as the existing ones weren't suffice

Reply:
Oh, actually, the returned data from go-buildkite isn't as detailed. I can raise a PR against go bk

Reply:
Ie it doesn't store a `Queues` field